extra/generate-display-names/display-names

```diff
@@ -137,6 +137,7 @@
 * ../../tables/ru-litbrl.ctb                     Russian                                            Russian braille
   ../../tables/ru-litbrl-detailed.utb            Russian, with capitals                             Russian braille with indication of capitals
   ../../tables/ru-compbrl.ctb                    Russian, for program sources                       Russian braille for program sources
+* ../../tables/ru-ru-g1.ctb                     Russian, grade 1                                            Russian contracted braille
```

Comment:
The indentation of the columns is not quite right, which might be the reason the tests are failing. I can fix it.

Reply:
I fixed this locally.